sdk/schemaregistry/schema-registry-avro/src/schemaRegistryAvroSerializer.ts

```diff
@@ -130,7 +131,9 @@ export class SchemaRegistryAvroSerializer {
    * @param buffer - The buffer with the serialized value.
    * @returns The deserialized value.
    */
-  async deserialize(buffer: Buffer): Promise<unknown> {
+  async deserialize(input: Buffer | Blob | Uint8Array): Promise<unknown> {
+    const arr8 = await toUint8Array(input);
+    const buffer = Buffer.from(arr8);
```

Comment:
this seems somewhat inefficient when a Buffer is passed in. Can't we use `Buffer.isBuffer` to disambiguate a Buffer from an `Uint8Array`?  Then we don't have to worry about having a node version of `toUint8Array` which doesn't do anything

Reply:
Right, done.